src/components/Dashboard/TodoList.vue

```diff
@@ -895,14 +897,16 @@ export default {
       })
     },
 
-    async doDeleteDraft (task) {
+    async doDeleteDraft (task, refreshDashboard: boolean = true) {
       const id = this.entityIncNo || this.tempRegNumber
       let url = `businesses/${id}/filings/${task.id}`
       await axios.delete(url).then(res => {
         if (!res) { throw new Error('Invalid API response') }
 
-        // emit dashboard reload trigger event
-        this.$root.$emit('triggerDashboardReload')
+        if (refreshDashboard) {
+          // emit dashboard reload trigger event
+          this.$root.$emit('triggerDashboardReload')
```

Comment:
Interesting! Is this an alternative to @Emit property decorator?

Reply:
I believe the emit decorator just emits to the parent, while the $root emit will send it to the root. In App.vue we listen on this event.

Reply:
oh thats pretty cool! Right on, thanks :)